Jellyfin.Plugin.Tvdb/Providers/TvdbExternalUrlProvider.cs

```diff
@@ -39,13 +44,14 @@ public IEnumerable<string> GetExternalUrls(BaseItem item)
 
                     break;
                 case Season season:
-                    season.Series.ProviderIds.TryGetValue(TvdbPlugin.SlugProviderId, out var seriesSlugId);
-                    var displayOrder = season.Series.DisplayOrder;
-                    if (string.IsNullOrEmpty(displayOrder))
+                    if (season.Series == null || season.Series.ProviderIds == null)
                     {
-                        displayOrder = "official";
+                        yield break;
                     }
 
+                    season.Series.ProviderIds.TryGetValue(TvdbPlugin.SlugProviderId, out var seriesSlugId);
+                    var displayOrder = season.Series.DisplayOrder ?? "official";
```

Comment:
?? is not the same is `IsNullOrEmpty`

Reply:
You're right. If `season.Series.DisplayOrder` were null, we assign "official". But "" displayorder might well be undefined, and thus a default would apply? I've updated it to handle `IsNullOrEmpty`.